soda/scientific/setup.py

```diff
@@ -18,14 +18,7 @@
     "inflection==0.5.1",
     "httpx>=0.18.1,<2.0.0",
     "PyYAML>=5.4.1,<6.0.0",
-    # The following incantation and Python 3.8 is neded to make stuff work.
-    # Sometimes in life, universe (by that I mean python) throws sh*t at you
-    # and you think you can reason with Python and PiP and try to solve a "problem"
-    # ... but you would be wrong.
-    # The best course of action is acceptance, despite the horrors you see in setup.py files
-    # such as this. And that my dear reader of the code is the real 'Zen of Python`.
-    "prophet @ git+https://github.com/facebook/prophet.git#egg=prophet&subdirectory=python",
-    # "prophet",
+    "prophet",
```

Comment:
I think we might want to require that it pins to at least `>=1.0.0` to be on the safe side. I think version prior to one have a completely different package name...
```suggestion
    "prophet>=1.0.0",
```